lib/logstash/outputs/elasticsearch/common.rb

```diff
@@ -16,6 +17,7 @@ module Common
 
     def register
       @stopping = Concurrent::AtomicBoolean.new(false)
+      @dlq_writer = LogStash::Util::DeadLetterQueueFactory::get(self)
```

Comment:
To support bwc, this should check if DLQ even exists. If it doesn't, we could provide a null dlq instance (perhaps one implemented in this plugin so it can work in older Logstash versions)